### ISSUE TITLE
Fix google sheets embed

### ISSUE THREAD

**Issue by kbourbery** [Watt-Esports/wattesports.co.uk#33]
Me and Nico currently cannot find a way to have the embed scale rather than be a fixed size, and the embed also removes the font in both the sheets document, as well as the styling font.
https://docs.google.com/spreadsheets/d/1TCX7HXFHgGsKgle7yqLouJugLmjBB0Rhh3Ucd5gjeSI/edit?usp=sharing

**Comment by KommToby** [comment]
I'm pretty sure you can only scale the jframe, but not the sheet itself because of how google exports the document

The idea I was going to have is that we could ping the google sheets api, collect the data and format it into a table we make ourselves, It would still collect all of the data from the google sheets, but we would be putting it into our own table instead of just embedding the google sheets table.

Perhaps we should go this direction instead?

**Comment by Danial-Q** [comment]
Yeah I can help with that idea if you wanna go down that route, I've worked with the google sheets api for the bot and can do it here

**Comment by KommToby** [comment]
> Yeah I can help with that idea if you wanna go down that route, I've worked with the google sheets api for the bot and can do it here

That would be really helpful, I honestly believe this is the easiest way to do this
